inst/examples/showcase/app.R

```diff
@@ -81,11 +78,26 @@ items <- do.call(c, lapply(sections, `[[`, "items"))
 
 makeNav <- function(sections) {
   lapply(sections, function(section) {
+    sectionId <- paste0(
+      "section-",
+      gsub(" ", "-", tolower(section$name))
+    )
     tagList(
-      H6(section$name),
-      UL(lapply(section$items, function(item) {
+      tags$button(
+        section$name,
+        class = "section-button",
+        onclick = paste0(
+          "$('#", sectionId, "').slideToggle();",
+          "$(this).toggleClass('expanded');"
+        )
+      ),
+      UL(
+        id = sectionId,
+        style = "display: none;",
+        lapply(section$items, function(item) {
         tags$li(
-          tags$a(item$name, href = route_link(item$id))
+          tags$a(item$name, href = route_link(item$id)),
+          class = "li-button"
         )
       }))
```

Comment:
Indentation is slightly off, consider this:
```r
      UL(
        id = sectionId,
        style = "display: none;",
        lapply(section$items, function(item) {
          tags$li(
            tags$a(item$name, href = route_link(item$id)),
            class = "li-button"
          )
        })
      )
```

Reply:
Fixed in #56